scripts/heat/instances.yaml

```diff
@@ -39,6 +39,16 @@ resources:
       flavor: { get_param: flavor }
       networks:
         - port: { get_resource: server_port }
+      user_data_format: RAW
+      user_data:
+        #!/bin/sh
+        cat >> /usr/bin/ping_forever << EOF
+        #!/bin/sh
+        trap "" 1 2 3 15
+        ping 8.8.8.8 | tee /var/log/ping_outside.out
```

Comment:
probably so: deploy a VM with heat, fetch the VIP (I think you can do that with heat too) and then ping it from the admin node.